src/js/client/main.js

```diff
@@ -82,6 +82,10 @@ $(document).ready(function () {
     });
   });
 
+  $(window).scroll(() => {
+    document.querySelector('.landing-welcome-background-box1').style.setProperty('--diamond-size', `${90 + window.scrollY / 14}vh`);
```

Comment:
This is a really cool way of achieving animations more declaratively.

The animation is a bit choppy - could we move this into a throttled `requestAnimationFrame` loop?